hana/exporter.sls

```diff
@@ -22,15 +23,11 @@ configure_exporter:
       - hanadb_exporter
       - python3-PyHDB
 
-stop_exporter:
-  process.absent:
-    - name: hanadb_exporter -c {{ config_file }} -m /etc/hanadb_exporter/metrics.json
-    - require:
-        - configure_exporter
-
 start_exporter:
-  cmd.run:
-    - name: nohup hanadb_exporter -c {{ config_file }} -m /etc/hanadb_exporter/metrics.json &>/dev/null &
+  service.running:
```

Comment:
in theory we could also `retry` the service start .. but imho we can just avoid for moment  this :grin:

Reply:
resolving